src/backend/distributed/utils/citus_depended_object.c

```diff
@@ -33,6 +33,8 @@
 #include "distributed/citus_depended_object.h"
 #include "distributed/metadata_cache.h"
 #include "distributed/listutils.h"
+#include "distributed/log_utils.h"
```

Comment:
why do we need to include log_utils.h?

Reply:
We use the GUC EnableUnsupportedFeatureMessages which is declared there.